Node-DC-EIS-client/runspec.py

```diff
@@ -68,6 +68,7 @@
 urllist= []
 memstat_interval = 3
 memlogfile = "memlog_file"
+show_graph = 1 #if set to 0, output graphs will not be generated.
```

Comment:
It seems kind of odd to create an option in which you pass 0 or 1 to enable and disable the graphs. It might be better to default to enabled and have an option (with action='store_false') called '--no-graphs'. Or default to disabled and have an option (with action='store_true') called '--graphs'. --no-graphs might be better since we are currently creating them.

Also, normally you don't use 0 and 1 for booleans in Python. Please switch to False and True.